noggin/templates/user-settings-otp.html

```diff
@@ -114,6 +114,9 @@ <h5 id="pageheading">{{ _("OTP Tokens") }}</h5>
         $('#otp-toggle').click(function() {
           $('#otp-qrcode').slideToggle("fast");
         })
+        $('#otp-modal').on("hide.bs.modal", function (e) {
+          if(!confirm("{{_('Are you sure you want to close the OTP dialog? This code will not be shown again, and you will be locked out of your account if you have no way to generate your OTP')}}")) return false;
```

Comment:
Instead of doing `if (!something()) return false;` I suppose you could just do `return something();`. Or if you really worry about types, you could do `return !!something();`. It looks ugly but it's apparently the standard way to cast to bool in javascript.